client/app.ts

```diff
@@ -1,31 +1,37 @@
-import { sendMessage } from "./server.js";
+import { sendAction, sendChatMessage } from './server.js';
+import {
+    chatData,
+    formatMessage,
+    printMessage,
+    initializeChatListener,
+} from './chat.js';
 declare var io: any;
 
 type Canvas = {
-    canvas: HTMLCanvasElement,
-    ctx: CanvasRenderingContext2D
-}
+    canvas: HTMLCanvasElement;
+    ctx: CanvasRenderingContext2D;
+};
 
-type Action = "Up" | "Down" | "Left" | "Right";
+type Action = 'Up' | 'Down' | 'Left' | 'Right';
```

Comment:
Autoformat also did this. Do you have a preference on single or double quotes? I can change my settings.

Reply:
I personally prefer double quotes, but nbd either way.

Reply:
ok, I'll update it and push the changes.